app/src/main/java/chat/rocket/android/api/MethodCallHelper.java

```diff
@@ -278,6 +278,15 @@ public MethodCallHelper(RealmHelper realmHelper, DDPClientWraper ddpClient) {
         .onSuccessTask(task -> Task.forResult(null));
   }
 
+  public Task<Void> setUserAway() {
+    return call("UserPresence:away", TIMEOUT_MS, JSONArray::new)
```

Comment:
No, please.. don't "fix" it.

I'll grab my computer on Monday and then we talk about it.